packages/contracts/contracts/SmartInvoiceEscrow.sol

```diff
@@ -37,20 +37,40 @@ contract SmartInvoiceEscrow is
         ARBITRATOR
     }
 
-    /// @notice Contract version identifier
-    string public constant VERSION = "3.0.0";
+    /// @notice Round accounting adapted from Kleros ArbitrableProxy
+    struct Round {
+        mapping(uint256 => uint256) paidFees; // amount raised per ruling option in this round
+        mapping(uint256 => bool) hasPaid; // true if that ruling option reached its specificCost in this round
+        mapping(address => mapping(uint256 => uint256)) contributions; // user => ruling => amount
+        uint256 feeRewards; // reimbursable appeal fees available to winners’ contributors in this round
+        uint256[] fundedRulings; // ruling options that fully funded in this round
+    }
 
-    /// @notice Number of ruling options available to arbitrators (client wins or provider wins)
+    /// @notice Single dispute bookkeeping
+    struct DisputeData {
+        bool isRuled; // whether a dispute is currently ruled
+        uint256 ruling; // final ruling
+        uint256 disputeId; // external dispute id
+    }
+
+    /// @notice Number of ruling options available to arbitrators: client vs provider; plus 0 = "refused/split"
     uint256 public constant NUM_RULING_OPTIONS = 2;
 
-    /// @notice Maximum allowed termination time (730 days from contract creation)
+    /// @notice Maximum allowed termination time (2 years from contract creation)
     uint256 public constant MAX_TERMINATION_TIME = 730 days;
 
     /// @notice Maximum number of milestones allowed per contract
     uint256 public constant MAX_MILESTONE_LIMIT = 50;
 
     uint256 internal constant BPS_DENOMINATOR = 10_000;
 
+    uint256 internal constant DISPUTE_EVIDENCE_GROUP_ID = 0;
+
+    /// @notice Appeal stake multipliers (basis points), aligned with ArbitrableProxy defaults
+    uint256 internal constant WINNER_STAKE_MULTIPLIER = 10000; // 1x appeal fee
+    uint256 internal constant LOSER_STAKE_MULTIPLIER = 20000; // 2x appeal fee
```

Comment:
These values are not mandatory and you can choose different ones if you want, but usually the defaults are `3000` for winner and `7000` for loser